Description: Don't set X-GNOME-Autostart-Phase=Panel to avoid delaying other softwares start on login
Author: ?

diff -Nur gnome-panel-2.29.91/gnome-panel/gnome-panel.desktop.in.in gnome-panel-2.29.91.ubuntu/gnome-panel/gnome-panel.desktop.in.in
--- gnome-panel-2.29.91/gnome-panel/gnome-panel.desktop.in.in	2010-02-16 19:58:31.000000000 +0100
+++ gnome-panel-2.29.91.ubuntu/gnome-panel/gnome-panel.desktop.in.in	2010-02-23 15:03:14.000000000 +0100
@@ -14,6 +14,6 @@
 OnlyShowIn=GNOME;
 NoDisplay=true
 X-GNOME-AutoRestart=true
-X-GNOME-Autostart-Phase=Panel
+#X-GNOME-Autostart-Phase=Panel
 X-GNOME-Provides=panel
 X-GNOME-Autostart-Notify=true
